<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-09-03T17:12:52Z" name="Default suite" finished-at="2024-09-03T17:12:52Z" duration-ms="137">
    <groups>
    </groups>
    <test started-at="2024-09-03T17:12:52Z" name="Default test" finished-at="2024-09-03T17:12:52Z" duration-ms="137">
      <class name="com.test4.Depends">
        <test-method signature="tc1()[pri:0, instance:com.test4.Depends@5f354bcf]" started-at="2024-09-03T17:12:52Z" name="tc1" finished-at="2024-09-03T17:12:52Z" duration-ms="30" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc1 -->
        <test-method signature="tc3()[pri:0, instance:com.test4.Depends@5f354bcf]" started-at="2024-09-03T17:12:52Z" name="tc3" finished-at="2024-09-03T17:12:52Z" duration-ms="18" status="FAIL">
          <exception class="junit.framework.AssertionFailedError">
            <message>
              <![CDATA[failure]]>
            </message>
            <full-stacktrace>
              <![CDATA[junit.framework.AssertionFailedError: failure
at junit.framework.Assert.fail(Assert.java:57)
at junit.framework.Assert.assertTrue(Assert.java:22)
at com.test4.Depends.tc3(Depends.java:22)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- junit.framework.AssertionFailedError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc3 -->
        <test-method signature="tc2()[pri:0, instance:com.test4.Depends@5f354bcf]" started-at="2024-09-03T17:12:52Z" name="tc2" finished-at="2024-09-03T17:12:52Z" depends-on-methods="com.test4.Depends.tc3" duration-ms="4" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc2 -->
      </class> <!-- com.test4.Depends -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
